src/middleware/logger.js

```diff
@@ -1,6 +1,13 @@
+import Log from '../models/log';
+
 export default (message) => {
     return (req, res, next) => {
         console.log(`Logging ${message}`);
+        new Log({
+            ip: req.ip,
+            method: req.method,
+            requested: `${req.protocol}://${req.hostname}${req.path}`
+        }).save();
```

Comment:
we should be calling `next()` on success or failure. aka, calls to next should be in `.save().then()` and `.save().catch()`